src/workerd/server/tests/python/import_tests.bzl

```diff
@@ -0,0 +1,45 @@
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("//:build/wd_test.bzl", "wd_test")
+load("//:build/pyodide_bucket.bzl", "PYODIDE_IMPORTS_TO_TEST")
+
+def generate_import_py_file(imports):
+  res = "def test():\n"
+  for imp in imports:
+    res += "   import "+imp+"\n"
+  return res
+
+def generate_wd_test_file(requirement):
+  return """
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "python-import-{}",
+      worker = (
+        modules = [
+          (name = "worker.py", pythonModule = embed "./worker.py"),
+          (name = "{}", pythonRequirement = ""),
+        ],
+        compatibilityDate = "2024-05-02",
+        compatibilityFlags = ["python_workers"],
+      )
+    ),
+  ],
+);""".format(requirement, requirement)
+
+def gen_import_tests():
+  for lib in PYODIDE_IMPORTS_TO_TEST.keys():
```

Comment:
maybe pass PYODIDE_IMPORTS_TO_TEST as a parameter? from BUILD file? It is better if rules don't depend on setup details.